.github/workflows/ci.yml

```diff
@@ -195,6 +181,18 @@ jobs:
             package-release.zip
             modules/sentry-java/sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib/*
 
+      # TODO: can we make unity only prepare the project, without compiling it, to unlock the unity license faster? iOS is already built like that.
```

Comment:
We can. We'll need to "export gradle project" on Android instead of building it. It makes sense to go that route

Reply:
not a new todo, just moved a couple of lines. 

And I'm aware it should work but didn't get around to do it